Tests/test_file_palm.py

```diff
@@ -2,15 +2,10 @@
 import subprocess
 
 import pytest
+
 from PIL import Image
```

Comment:
```suggestion
import pytest
from PIL import Image
```
I find that removing the blank line fixes isort